posthog/hogql/database/schema/sessions_v2.py

```diff
@@ -37,6 +37,9 @@
 if TYPE_CHECKING:
     from posthog.models.team import Team
 
+
+DEFAULT_BOUNCE_RATE_DURATION_SECONDS = 30
```

Comment:
Default was actually 10 if you look at the old line 274

Reply:
I should not try to work on a train 🙃